client/src/components/Workflow/Editor/Node.vue

```diff
@@ -314,6 +314,7 @@ export default {
         },
         setData(data) {
             this.config_form = data.config_form;
+            this.content_id = data.config_form?.id || data.content_id || this.content_id;
```

Comment:
Thanks a lot @mvdbeek. Alternatively, we could consistently return the `content_id` with the API like here: https://github.com/galaxyproject/galaxy/commit/c46653cdd575725f5243b7c8532b7ae003f00037.

Reply:
Yeah, I think that's consistent, I'll pick that commit

Reply:
Hmm, that didn't quite work https://github.com/galaxyproject/galaxy/runs/4868498354?check_suite_focus=true#step:7:2323